tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/merge.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import PageRegion
+
+
+class Merge(PageRegion):
```

Comment:
Not sure about using the page region here since during the test dealing with the iframe seemed to me different as with other elements. I cannot access, or I do not know how, the elements in it as I do in with other elements. So in the tap_on_merge method I cannot do: self.root_element.....the only way I got it working is as it is now...:(